sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/FeatureLayer.cs

```diff
@@ -262,12 +267,58 @@ private void MoveCamera()
         var index = GetAllFeatures ? 0 : StartValue;
         var cameraLocationComponent = arcGISCamera.gameObject.GetComponent<ArcGISLocationComponent>();
         var position = new ArcGISPoint(FeatureItems[index].GetComponent<ArcGISLocationComponent>().Position.X,
-            FeatureItems[index].GetComponent<ArcGISLocationComponent>().Position.Y, 10000, cameraLocationComponent.Position.SpatialReference);
+            FeatureItems[index].GetComponent<ArcGISLocationComponent>().Position.Y, 500,
+            cameraLocationComponent.Position.SpatialReference);
         cameraLocationComponent.Position = position;
         cameraLocationComponent.Rotation = new ArcGISRotation(cameraLocationComponent.Rotation.Heading, 0.0,
             cameraLocationComponent.Rotation.Roll);
     }
 
+    public void RefreshProperties(GameObject featureItem)
+    {
+        FeatureQuery currentFeature = new FeatureQuery();
+        //Layer 7 because that is the index of the layer created specifically for feature layers so that they ignore themselves for raycasting.
+        featureItem.layer = 7;
+        featureInfo = featureItem.GetComponent<FeatureData>();
+        featureInfo.Properties.Clear();
+        featureInfo.Coordinates.Clear();
+        locationComponent = featureItem.GetComponent<ArcGISLocationComponent>();
+        var coordinates = jFeatures[featureInfo.Index].SelectToken("geometry").SelectToken("coordinates").ToArray();
+        var properties = jFeatures[featureInfo.Index].SelectToken("properties").ToArray();
+
+        if (GetAllOutfields)
+        {
+            foreach (var value in properties)
+            {
+                var key = value.ToString();
+                var props = key.Split(":");
+                currentFeature.properties.propertyNames.Add(props[0]);
+                currentFeature.properties.data.Add(props[1]);
+                featureInfo.Properties.Add(key);
+            }
+        }
+        else
+        {
+            for (var j = 0; j < outfields.Count; j++)
```

Comment:
replace `j` with something descriptive

Reply:
I don't think this is super necessary. Any user that wants to look at code and has knowledge of for loops will expect the variables to be i and j for each respective loop. Changing the variable names isn't going to make impact the sample or readability of the code significantly so I don't think it is necessary.